depthmapX/GraphDoc.cpp

```diff
@@ -312,6 +312,74 @@ void QGraphDoc::ProcPostMessage(int m, int x, int y)
     }
 }
 
+void QGraphDoc::OnVGALinksFileImport()
+{
+    if (m_communicator) {
+        return; // Locked
+    }
+
+    // change the view before loading the file to make the changes apparent
+    ((QDepthmapView*)m_view[VIEW_MAP])->m_showlinks = true;
+    SetRedrawFlag(VIEW_MAP,REDRAW_POINTS, NEW_DEPTHMAPVIEW_SETUP);
+
+    QString template_string;
+    template_string += "Text files (*.txt *.csv)\n";
+    template_string += "All files (*.*)";
+
+    QFileDialog::Options options = 0;
+    QString selectedFilter;
+    QStringList infiles = QFileDialog::getOpenFileNames(

```

Comment:
Why not use QFileDialog::getOpenFileName (not the singular) instead of checking that not more than one file was selected?